### ISSUE TITLE
Set FOB as Repair Facility for ACE_Repair

### ISSUE THREAD

**Issue by MoistBaguette** [fparma/liberation#25]

**Comment by Cuel** [comment]
Make sure everyone can use toolkits aswell

**Comment by MoistBaguette** [comment]
Ye, made sure about this.
6f141e4b9fd97810d844ebc0f07238481f5c31bb marks every player as an Engineer.

**Comment by Cuel** [comment]
I've set the default to allow everyone to repair in serverconfig
I guess you can still have everyone be engineer tho that makes no failed defuses, and everyone can repair vehicles 100%

**Comment by Cuel** [comment]
Yeah I changed the default yesterday, so everyone can repair now, if the server has been restarted
